### PR TITLE
MM-T506 Channel links show as links in notification emails

### PR DESCRIPTION
**Pull request by josephk96**
Summary

This test checks that when a user is tagged in a post that includes the channel name, the user receives an email with the expected/correct output.

Test case: https://automation-test-cases.vercel.app/test/MM-T506

### CODE REVIEW

File: e2e/cypress/integration/notifications/channel_links_show_as_links_spec.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+// ***************************************************************
+
+// Group: @notifications
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+import {getEmailUrl, getEmailMessageSeparator, reUrl} from '../../utils';
+const baseUrl = Cypress.config('baseUrl');
+const mailUrl = getEmailUrl(baseUrl);
+
+describe('Team Settings', () => {
```
Comment (srkgupta) [inline e2e/cypress/integration/notifications/channel_links_show_as_links_spec.js:18]: Suggested change:
```describe('Notifications', () => {```

**Review by srkgupta — APPROVED** [review]
Thanks @josephk96. The test looks awesome and well drafted. Great work. Just a very minor suggestion.

File: e2e/cypress/integration/notifications/channel_links_show_as_links_spec.js (hunk 1 above)
Comment (saturninoabril) [reply e2e/cypress/integration/notifications/channel_links_show_as_links_spec.js:18]: I agree
```suggestion
describe('Notifications', () => {
```

**Review by saturninoabril — APPROVED** [review]
Thanks @josephk96, LGTM!

**Pull request merged**